lib/mongodb.js

```diff
@@ -49,13 +49,17 @@ exports.generateMongoDBURL = generateMongoDBURL;
  * Generate the mongodb URL from the options
  */
 function generateMongoDBURL(options) {
+  // See https://docs.mongodb.com/manual/reference/connection-string/#dns-seedlist-connection-format
+  // It can be `mongodb+srv` now.
+  var protocol = options.protocol || 'mongodb';
```

Comment:
For consistency can we make this `options.protocol = options.protocol || 'mongodb'`